internal/coredns/coredns.go

```diff
@@ -49,6 +49,7 @@ const (
 `
 	hostsTemplate = `
     {{ .K3dRegistryIP}} {{ .K3dRegistryHost}}
+	{{ .K3dRegistryIP}} {{ .K3dRegistryHost}}.localhost
```

Comment:
would it be possible to do the localhost patching only on the alpha command. there are concerns in the team that we do not want localhost exposure for the current deploy.